src/serde/ser.rs

```diff
@@ -74,13 +67,15 @@ where
 /// Implements a standard serializer for Ion
 pub struct ValueSerializer<'a, V: ValueWriter> {
     pub(crate) value_writer: V,
+    pub(crate) is_human_readable: bool,
     lifetime: PhantomData<&'a ()>,
 }
 
 impl<'a, V: ValueWriter> ValueSerializer<'a, V> {
-    pub fn new(value_writer: V) -> Self {
+    pub fn new(value_writer: V, is_human_readable: bool) -> Self {
```

Comment:
Having this as a mandatory parameter on the constructor is a bit unfortunate, but I'm ok with it for now since we'll eventually be able to remove it when `LazyValue` is able to report its encoding.

Reply:
#793